File: codewars/8kyu/convert-a-number-to-a-string.py
# SOLUTION
# ==============================================================================
def number_to_string(num):
    return str(num)

# TESTS
# ==============================================================================
if __name__ == "__main__":
    assert number_to_string(123) == "123", "Test case 1 failed"
    assert number_to_string(0) == "0", "Test case 2 failed"
    assert number_to_string(-123) == "-123", "Test case 3 failed"
    assert number_to_string(999999) == "999999", "Test case 4 failed"
    assert number_to_string(-999999) == "-999999", "Test case 5 failed"
    print("All test cases passed!")

File: codewars/8kyu/convert-a-number-to-a-string.cpp
#include <iostream>
#include <string>
#include <cassert>

std::string number_to_string(int num) {
    return std::to_string(num);
}

void runTests() {
    assert(number_to_string(123) == "123");
    assert(number_to_string(999) == "999");
    assert(number_to_string(-100) == "-100");
    std::cout << "All tests passed!" << '\n';
}

int main() {
    runTests();
    return 0;
}
